src/components/PublicMessage.jsx

```diff
@@ -28,7 +28,10 @@ export default function PublicMessage({message, routes}) {
           ))}
         </th>
       )}
-      <td colSpan={(routes) ? 1 : 2}>
+      <td
+        className={`${classNames['priority']} ${classNames['priority-' + priority]}`}
```

Comment:
```suggestion
        className={`${classNames['priority']} ${classNames[`priority-${priority}`]}`}
```
interpo-ception

Reply:
doesn't work here for whatever reason, it's deceptive because the default color matches the color of all of the messages right now, but if you look at the rendered html, this is what happens:
```html
<td class="_priority_1yyjt_14 undefined" colspan="1">Until further notice, B79 service may use vans for some service. &lt;a href='https://www.pvta.com/info.php'&gt;Click here&lt;/a&gt; for more information.</td>
```